docs/howto/openstack/octavia/metrics.md

```diff
@@ -0,0 +1,124 @@
+---
+description: How to add a Prometheus metrics endpoint to a load balancer.
+---
+# Enabling load balancer metrics
+
+> This feature is available in select {{brand}} regions.
+> Please check the [feature support matrix](../../../reference/features/index.md) for details.
+
+Once you have created load balancers in {{brand}}, you can add a special *listener* that exposes load balancer metrics.
+This listener is meant to be used as a data source for an existing [Prometheus](https://prometheus.io/) monitoring system.
+
+## Prerequisites
+
+The {{gui}} does not support defining metrics endpoints, so you will have to work with the OpenStack CLI.
+[Enable it](../../getting-started/enable-openstack-cli.md) for the region you will be working in, and make sure you have the Python `octaviaclient` package installed.
+For that, use either the package manager of your operating system, or `pip`:
+
+=== "Debian/Ubuntu"
+    ```bash
+    apt install python3-octaviaclient
+    ```
+=== "Mac OS X with Homebrew"
+    This particular Python module is unavailable via `brew`, but you
+    can install it via `pip`.
+=== "Python Package"
+    ```bash
+    pip install python-octaviaclient
+    ```
+
+## Assumptions and scenario
+
+We assume you already have an [HTTPS-terminated load balancer](tls-lb.md) that forwards client requests to a back-end server pool.
+In our test scenario, the load balancer was accepting HTTPS requests for `whoogle.example.com` and forwarding them to a two-member pool, with servers each running a Docker container for [Whoogle Search](https://github.com/benbusby/whoogle-search).
+
+## Creating a metrics listener
+
+Here is `mylb`, the load balancer we used during testing...
```

Comment:
```suggestion
Here is `mylb`, the load balancer we used during testing:
```
(I use the dots when presenting more than one console output box, in the context of a long sentence.)

Reply:
Point taken (see what I did there?)